components/RefinementList.js

```diff
@@ -19,7 +19,10 @@ class RefinementList extends React.Component {
   // has a checkbox inside, we ignore the first click event because we will get another one.
   handleClick(value, e) {
     if (e.target.tagName === 'A' && e.target.href) {
+      // do not trigger any url change by the href
       e.preventDefault();
+      // do not bubble (so that hierarchical lists are not triggering refine twice)
+      e.stopPropagation();
```

Comment:
We will probably never do it. 



—
Alexandre STANISLAWSKI 
Software engineer 

Phone: +33 6 32 67 05 12 
Website: https://www.algolia.com/

On Thu, Oct 1, 2015 at 5:10 PM, Vincent Voyer <notifications@github.com>
wrote:

>>        e.preventDefault();
>> +      // do not bubble (so that hierarchical lists are not triggering refine twice)
>> +      e.stopPropagation();
> I dunno if we need this **right** now
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/algolia/instantsearch.js/pull/149/files#r40924500